README.md

```diff
@@ -57,205 +56,136 @@ oc cluster up
 
 More information about this way can be found [here][occlusterup].
 
-## Kafka Stateful Sets
+## Cluster Controller
 
-This deployment uses the StatefulSets (previously known as "PetSets") feature of Kubernetes/OpenShift. 
-With StatefulSets, the pods receive a unique name and network identity and that makes it easier to identify the 
-individual Kafka broker pods and set their identity (broker ID). 
-Each Kafka broker pod is using its own PersistentVolume. 
-The PersistentVolume is acquired using PersistentVolumeClaim – that makes it independent on the actual type of the PersistentVolume. 
-For example, it can use HostPath volumes on Minikube or Amazon EBS volumes in Amazon AWS deployments without any changes in the YAML files.
-
-It's important to say that in this deployment both _regular_ and _headless_ services are used:
-
-* regular services can be used as bootstrap servers for Kafka clients;
-* headless services are needed to have DNS resolve the pods IP addresses directly.
-
-This deployment is available under the _kafka-statefulsets_ folder and provides following artifacts:
-
-* Dockerfile : Docker file for building an image with Kafka and Zookeeper already installed
-* config : configuration file templates for running Zookeeper
-* scripts : scripts for starting up Kafka and Zookeeper servers
-* resources : provides all YAML configuration files for setting up volumes, services and deployments
+Strimzi is using process called Cluster Controller to deploy and manage Kafka and Kafka Connect clusters. Cluster Controller is a process which is running inside your Kubernetes or OpenShift cluster. To deploy a Kafka cluster, a ConfigMap with the cluster configuration has to be created. The ConfigMap needs ot be labaled with followign labels:
```

Comment:
do we want to add something like "with related Zookeeper ensemble" or something like that ?

Reply:
There are also three typos in a row, congratulations !! :-) "...needs ot be labaled with followign ..." instead of " ... needs to be labeled with following ..."